### PR TITLE
[Arm64] Fix #17095

### LINKED ISSUE #17095: Arm64 Ubuntu CI test failures
The debug/checked/release "small page size" test jobs all fail with:
```
21:55:10 + cp ./bin/Product/Linux.arm64.Debug/corefxNative/System.IO.Compression.Native.a ./bin/Product/Linux.arm64.Debug/corefxNative/System.IO.Compression.Native.so ./bin/Product/Linux.arm64.Debug/corefxNative/System.Native.a ./bin/Product/Linux.arm64.Debug/corefxNative/System.Native.so ./bin/Product/Linux.arm64.Debug/corefxNative/System.Net.Http.Native.a ./bin/Product/Linux.arm64.Debug/corefxNative/System.Net.Http.Native.so ./bin/Product/Linux.arm64.Debug/corefxNative/System.Net.Security.Native.a ./bin/Product/Linux.arm64.Debug/corefxNative/System.Net.Security.Native.so ./bin/Product/Linux.arm64.Debug/corefxNative/System.Security.Cryptography.Native.OpenSsl.a ./bin/Product/Linux.arm64.Debug/corefxNative/System.Security.Cryptography.Native.OpenSsl.so ./bin/CoreFxBinDir
21:55:10 cp: target './bin/CoreFxBinDir' is not a directory
21:55:10 
Build step 'Execute shell' marked build as failure
```

e.g.:
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_debug_small_page_size_tst/845/console
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_release_small_page_size_tst/818/console
https://ci.dot.net/job/dotnet_coreclr/job/master/job/arm64_checked_small_page_size_tst/810/console

### PR DESCRIPTION
**Pull request by sdmaclea**
Fixes #17095 

@BruceForstall @jashook @mmitche @weshaggard PTAL
@dotnet/arm64-contrib @RussKeldorph FYI

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -2861,6 +2861,7 @@ Constants.allScenarios.each { scenario ->
                                 // HACK -- Arm64 does not have corefx jobs yet.
                                 // Clone corefx and build the native packages overwriting the x64 packages.
                                 if (architecture == 'arm64') {
+                                    shell("mkdir -p ./bin/CoreFxNative")
```
Comment (sdmaclea) [inline netci.groovy]: Direct copy from below
https://github.com/dotnet/coreclr/blob/235a34673994560deb6011be08aff18b4399658b/netci.groovy#L2867-L2868

**Comment by sdmaclea** [comment]
IF you wan to test it before merge someone with permissions needs to trigger "test ci please" then trigger the newly created jobs (after enabling them?)

File: netci.groovy (hunk 1 above)
Comment (BruceForstall) [reply netci.groovy]: You got the path wrong: it should be "./bin/CoreFxBinDir" to match the copy below.

Comment (sdmaclea) [reply netci.groovy]: Thanks.  I assumed I wrote the code and would have chosen the same name...  Blind....

Comment (sdmaclea) [reply netci.groovy]: Done

**Comment by BruceForstall** [comment]
@dotnet-bot test CI please

**Comment by sdmaclea** [comment]
Has someone run the generated arm64 jobs?  Can this be merged?

**Comment by BruceForstall** [comment]
Sorry for the delay. I'll merge it, since the CI jobs build worked. We should kick off the failing jobs (or wait for them to be automatically kicked off) to see if it really fixes the problem. (Unfortunately, submitting netci.groovy PR doesn't / can't actually run the changed jobs)

**Pull request merged**